cmd/gateway-main.go

```diff
@@ -112,6 +112,14 @@ func StartGateway(ctx *cli.Context, gw Gateway) {
 		cli.ShowCommandHelpAndExit(ctx, gatewayName, 1)
 	}
 
+	// Get "json" flag from command line argument.
+	// Set quiet mode if json flag is set.
+	jsonFlag := ctx.Bool("json") || ctx.GlobalBool("json")
+	if jsonFlag {
+		log.EnableJSON()
+		log.EnableQuiet()
```

Comment:
This is incorrect way of address.  `EnableJSON()` should do `EnableQuiet()` not gateway/server/something else